packages/renderer/MdToHtml/renderMedia.ts

```diff
@@ -44,16 +46,30 @@ export default function(link: Link, options: Options) {
 	}
 
 	if (options.pdfViewerEnabled && resource.mime === 'application/pdf') {
+
 		if (options.useCustomPdfViewer) {
+			const resourceId = resource.id;
 			let anchorPageNo = null;
+
+			let id = `${options.noteId}.${resourceId}`;
+			if (linkIndexes && linkIndexes[resourceId]) {
+				linkIndexes[resourceId]++;
+			} else {
+				linkIndexes[resourceId] = 1;
+			}
+			id += `.${linkIndexes[resourceId]}`;
+
 			if (link.href.indexOf('#') > 0) {
 				anchorPageNo = Number(link.href.split('#').pop());
 				if (anchorPageNo < 1) anchorPageNo = null;
 			}
-			return `<iframe src="../../vendor/lib/@joplin/pdf-viewer/index.html" url="${escapedResourcePath}" 
-			appearance="${options.theme.appearance}" ${anchorPageNo ? `anchorPage="${anchorPageNo}"` : ''}
+
+			return `<iframe src="../../vendor/lib/@joplin/pdf-viewer/index.html" x-url="${escapedResourcePath}" 
```

Comment:
Can you use `bridge().vendorDir()` for this path? Or pass vendorDir as a variable to this renderMedia function.